### PR TITLE
Publish fr/lecons/analyse-reseau-python

### LINKED ISSUE #3246: Preparing to publish a new FR translation analyse-reseau-python
I am opening this Issue to prepare a new FR translation for publication.

### PR DESCRIPTION
**Pull request by anisa-hawes**
Preparing files for publication.

Closes #3246

### Checklist

- [x] Assign yourself in the "Assignees" menu
- [x] Add the appropriate "Label"
- [x] If this PR closes an Issue, add the phrase `Closes #ISSUENUMBER` to your summary above
- [x] Ensure the status checks pass: if you have difficulty fixing build errors, please contact our Publishing Manager @anisa-hawes 
- [x] Check the Netlify Preview: navigate to netlify/ph-preview/deploy-preview and click 'details' (at right)
- [x] Assign at least one individual or team to "Reviewers"
  - ~~[ ] if the text needs to be translated, please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines), then assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing editor in your PR.~~

### CODE REVIEW

**Comment by anisa-hawes** [comment]
Bonjour Marie, @marie-flesch.

I’ve staged analyse-reseau-python for publication. You can [read it through in Preview](https://deploy-preview-3247--ph-preview.netlify.app/) and let me know if you note anything which you'd like to adjust.

If you're happy, we can organise to publish it when our colleagues at Sussex will be around to activate our DOI, and then we can launch the promotions on social media.

--

Thank you,
A.

File: fr/lecons/analyse-reseau-python.md [hunk 1]
```diff
@@ -0,0 +1,574 @@
...
+```
+Source,Target
+George Keith,Robert Barclay
+George Keith,Benjamin Furly
+George Keith,Anne Conway Viscountess Conway and Killultagh
+George Keith,Franciscus Mercurius van Helmont
+George Keith,William Penn
+```
+
+Maintenant que vous avez téléchargé les données Quaker et vu comment elles sont structurées, il est temps de commencer à travailler avec ces données à l'aide de Python. Une fois Python et pip installés (voir [Prérequis](#prérequis) ci-dessus), vous devez installer NetworkX, en tapant ceci dans votre ligne de commande (voir notre leçon disponible [en anglais](/en/lessons/intro-to-bash) et [en espagnol](/es/lecciones/introduccion-a-bash))[^4]&nbsp;:
```
Comment (marie-flesch) [inline fr/lecons/analyse-reseau-python.md]: At the end of ligne 108, we need to remove the link to the EN and ES lessons and add an FR link, as the FR lesson about Bash was just published :), like so:

(voir [notre leçon sur les interfaces en ligne de commande](/fr/lecons/intro-a-bash-et-zsh)[^4]&nbsp;:

**Comment by anisa-hawes** [comment]
Hello again @marie-flesch,

I noted a small error in my last commit (a missing closing bracket) but upon fixing that, I've encountered a handful of broken links elsewhere on the site... I've opened #3252 to investigate and find replacements. I'll rebase this PR when that work is complete. Apologies for this delay!

Anisa

**Comment by marie-flesch** [comment]
No worries @anisa-hawes, thank you!

**Comment by anisa-hawes** [comment]
Bonsoir @marie-flesch,

The broken links which caused problems earlier on are now back online. Apologies again for this delay.

I've adjusted the `translation_date:` to tomorrow. If you're happy with everything, we can publish it in the morning! ☺️

**Review by marie-flesch — APPROVED** [review]

**Pull request merged**